code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -240,6 +240,35 @@ GLOBAL_LIST_INIT(wood_recipes, list ( \
 	novariants = TRUE
 	grind_results = list("carbon" = 20)
 
+
+/obj/item/stack/sheet/mineral/wood/attackby(obj/item/W, mob/user, params) // NOTE: sheet_types.dm is where the WOOD stack lives. Maybe move this over there.
+	// Taken from /obj/item/stack/rods/attackby in [rods.dm]
+	if(W.get_sharpness())
+		user.visible_message("[user] begins whittling [src] into a pointy object.", \
+				 "<span class='notice'>You begin whittling [src] into a sharp point at one end.</span>", \
+				 "<span class='italics'>You hear wood carving.</span>")
+		// 8 Second Timer
+		if(!do_mob(user, src, 80))
```

Comment:
do_mob is used when both the user and the target are mobs. I guess this should be
```suggestion
		if(!do_after(user, 80, TRUE, src))
```
Though, currently, there is no such big difference between do_mob and do_after...